services/web/server/src/simcore_service_webserver/projects/_controller/comments_rest.py

```diff
@@ -1,6 +1,4 @@
-""" Handlers for project comments operations
-
-"""
+"""Handlers for project comments operations"""
```

Comment:
handlers? or controllers...

Reply:
@sanderegg The functions in a `controller` will still be called `handlers` :-) 
That said, this is also an old comment and I see they might lead to confusion so I will remove them